### ISSUE TITLE
Genate schemas with absolute path while using DaoGenerator.

### ISSUE THREAD

**Issue by CoderYellow** [ThirtyDegreesRay/OpenHub#180]
I think this [local path](E:/Work/Android/github/OpenHub/OpenHub/app/src/main/java) might only works on your computer.Maybe you should change to the relative path.

**Comment by CodeFarmer1995** [comment]
@CoderYellow I don't think so. He had generated the files once on this [path](https://github.com/ThirtyDegreesRay/OpenHub/tree/master/app/src/main/java/com/thirtydegreesray/openhub/dao). We don't have to do that again.
